app/src/models/InventoryValue.ts

```diff
@@ -238,6 +238,24 @@ export class InventoryValue
             fields: [{ name: "id" }],
           },
         ],
+        hooks: {
+          afterCreate: async (inventoryValue: InventoryValue) => {
+            if (inventoryValue.inventoryId) {
+              const inventory = await inventoryValue.getInventory();
+              if (inventory) {
+                await inventory.update({ lastUpdatedAt: new Date() });
+              }
+            }
+          },
+          afterUpdate: async (inventoryValue: InventoryValue) => {
+            if (inventoryValue.inventoryId) {
+              const inventory = await inventoryValue.getInventory();
+              if (inventory) {
+                await inventory.update({ lastUpdatedAt: new Date() });
```

Comment:
Can we just set lastUpdated here and add it to the model instead of lastUpdateAt if necessary, as the DB column exists already...